xml/deploy_cephadm.xml

```diff
@@ -643,4 +643,95 @@ config: OK
    </tip>
   </sect2>
  </sect1>
+ <sect1 xml:id="deploy-cephadm-day2">
+  <title>Further Deployment and Administration (Day 2)</title>
+
+<!-- 2020-04-20 tbazant: seems it boils down to:
+   * listing services/devices
+   * host manipulation
+   * deploy services
+  -->
+
+  <para>
+   After you deployed the basic &ceph; cluster, you need to deploy core
+   services on more cluster nodes. To make the cluster data accessible to
+   clients, deploy additional services.
+  </para>
+
+  <tip>
+   <title>&cephsalt; and <command>ceph orch</command> CLI</title>
+   <para>
+    While &cephsalt; helps you configure basic aspects of the cluster as
+    outlined in <xref linkend="deploy-cephadm-day1"/>, you will need it further
+    on either to modify the basic configuration or add a new host to the
+    cluster.
+   </para>
+   <para>
+    The &ceph; orchestrator command <command>ceph orch</command>&mdash;which is
+    an interface to the &cephadm; module&mdash;will take care of listing
+    cluster components and deploying &ceph; services on new cluster nodes.
+   </para>
+  </tip>
+
+  <sect2 xml:id="deploy-cephadm-day2-orch">
+<!-- https://docs.ceph.com/docs/master/mgr/orchestrator/ -->
+   <title>The <command>ceph orch</command> Command</title>
+   <para></para>
+  </sect2>
+
+  <sect2 xml:id="deploy-cephadm-day2-cephupgrade">
+<!-- https://docs.ceph.com/docs/master/cephadm/upgrade/ -->
+   <title>Upgrading &ceph;</title>
+   <para>
+    You can instruct &cephadm; to upgrade &ceph; from one bugfix release to
+    another. The automated upgrade of &ceph; services respects the recommended
+    order&mdash;it starts with &mgr;s, &mon;s, and then continues on other
+    services such as &osd;s, &mds;s, and &ogw;s. Each daemon is restarted only
+    after &ceph; indicates that the cluster will remain available.
+   </para>
+   <sect3 xml:id="deploy-cephadm-day2-cephupgrade-start">
+    <title>Starting the Upgrade</title>
+    <para>
+     Before you start the upgrade, verify that all hosts are currently online
+     and your cluster is healthy:
+    </para>
+<screen>&prompt.cephuser;ceph -s</screen>
+    <para>
+     To upgrade (or downgrade) to a specific &ceph; release:
+    </para>
+<screen>&prompt.cephuser;ceph orch upgrade start --ceph-version <replaceable>VERSION</replaceable></screen>
```

Comment:
```suggestion
<screen>&prompt.cephuser;ceph orch <replaceable>upgrade</replaceable> start --ceph-version <replaceable>VERSION</replaceable></screen>

Reply:
Since the comment above says "upgrade (or downground)" presumably the opposing command is upgrade v. downground as is here? If so, I think we should add the replaceable tags so the user can figure that one out easily :)

Reply:
there is no 'downgrade' subcommand, the 'upgrade' one either upgrades or downgrades to the specified version from the current version